hail/www/references.md

```diff
@@ -0,0 +1,90 @@
+
+# Scientific Output from the Hail Community  
+
+The following is a sample of research results that have used Hail as some part of the
```

Comment:
Suggested rewrite: The following is an incomplete list of scientific work enabled by Hail. We welcome you to add additional examples by [editting the page directly](https://github.com/hail-is/hail/edit/master/hail/www/references.md), after which we will review the pull request to confirm the addition is valid. Please adhere to the existing formatting conventions.